view/src/components/produce.js

```diff
@@ -230,10 +220,112 @@ class produce extends Component {
 		});
 	}
 
+	// This function updates the string that the produce array will be searched for.
+	// This string value can contain any produce field (name, weight, interrnal shipment numbers) and will still return
+	// 	the appropriate filtered results.
+
 	handleSearch = event => {
-		const { value } = event.target;
-		this.setState({ value });
-	  };
+    const { value } = event.target;
+    this.setState({ value });
+	};
+
+	// Handles the rendering of filtered produce results into React cards
+	handleResultsRender = (results) => {
+		const { classes } = this.props;
+		return (
+			<div>
+			{results.map((produce) => (
+			<Grid item xs={12} key={produce.produceId}>
+				<Card className={classes.root} variant="outlined">
+				<CardContent>
+					<Typography variant="h5" component="h2">
+					{produce.name}
+					</Typography>
+					<Box
+					display="flex"
+					flexDirection="row"
+					flexWrap="wrap"
+					p={0}
+					m={0}
+					>
+					<Box p={3}>
+						<Typography
+						className={classes.pos}
```

Comment:
what's the `.pos` here? maybe it should be spelled out?

Reply:
(switching it to position here, will roll it out incrementally in the other files as Andrew adjusts his own files)